tests/directives/test_item_attribute_directive.py

```
@@ -0,0 +1,319 @@
+        """Test run method with a defined attribute and caption argument"""
+        # Setup directive with caption
+        self.directive.arguments = ['test_attr_id', 'Test Caption']
+
+        # Setup defined attribute
+        attr_id = 'test_attr_id'
+        attr = TraceableAttribute(attr_id, '.*')
+        attr.name = 'Test Attribute'
+        TraceableItem.defined_attributes[attr_id] = attr
+
+        # Run the directive
+        result = self.directive.run()
```

Flake8 (7.3.0)

[error] local variable 'result' is assigned to but never used

(F841)